lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -7,7 +7,7 @@
   raise 'Could not load the braintree gem.  Use `gem install braintree` to install it.'
 end
 
-raise "Need braintree gem >= 3.0.0. Run `gem install braintree --version '~>3.0.0'` to get the correct version." unless Braintree::Version::Major == 3 && Braintree::Version::Minor == 0
+raise "Need braintree gem >= 2.0.0. Run `gem install braintree --version '~>3.0.0'` to get the correct version." unless Braintree::Version::Major >= 2 && Braintree::Version::Minor >= 0
```

Comment:
Oh wait @naashton can you update this comment to show '~> 2.0.0'?

Reply:
I actually think that 2nd sentence should just be removed. Thanks for pointing it out!

Reply:
👍🏻